include/rosdiscover-clang/BackwardSymbolizer/FunctionSymbolizer.h

```diff
@@ -888,7 +888,10 @@ class FunctionSymbolizer {
   std::unique_ptr<SymbolicCompound> symbolizeCompound(RawCompound *stmt) {
     auto result = std::make_unique<SymbolicCompound>();
     for (auto &s: stmt->getStmts()) {
-      result->append(symbolizeStatement(s));
+      auto symbolicStmt = symbolizeStatement(s);
+      if (symbolicStmt != nullptr) {
```

Comment:
It might be worth logging an error/warning if you weren't able to symbolize a given statement